docs/themes.md

```diff
@@ -1,7 +1,7 @@
 # Themes
 
 ```sh
-npm i @mdx-deck/themes
+npm i mdx-deck/themes
```

Comment:
Thanks for taking a stab at updating the docs! This isn't a valid package name, so I think this should be removed

Reply:
Indeed, that's my mistake. Now fixed with my latest commit.